test/Microsoft.IdentityModel.Protocols.OpenIdConnect.Tests/OpenIdConnectConfigurationTests.cs

```diff
@@ -345,5 +364,25 @@ public void NonemptyCollectionSerialization()
             }
             TestUtilities.AssertFailIfErrors(context);
         }
+
+        [Fact]
+        public void NonemptyCollectionSerializationWithStream()
```

Comment:
I think a "round trip" test is useful here. Write to a stream, read it, create the object, compare the properties.